src/sentry/static/sentry/app/components/externalIssues/abstractExternalIssueForm.tsx

```diff
@@ -87,12 +92,18 @@ export default class AbstractExternalIssueForm<
 
   onRequestSuccess = ({stateKey, data}) => {
     if (stateKey === 'integrationDetails') {
+      this.handleReceiveIntegrationDetails(data);
       this.setState({
         dynamicFieldValues: this.getDynamicFields(data),
       });
     }
   };
 
+  /**
+   * If this field should updateFrom, updateForm. Otherwise, do nothing.
+   * @param label
+   * @param value
```

Comment:
@mgaeta Is this for JSdoc? I think its unnecessary unless we add more details to this like https://github.com/getsentry/sentry/blob/efe9f9c470cdbee7bccb8c210d506e01f5af2937/src/sentry/static/sentry/app/actionCreators/events.tsx#L36